<inh f='include/c++/9/bits/exception.h' l='60' c='std::exception'/>
<def f='include/c++/9/ext/concurrence.h' l='72' ll='78'/>
<use f='include/c++/9/ext/concurrence.h' l='103' c='__gnu_cxx::__throw_concurrence_unlock_error'/>
<size>8</size>
<fun r='__gnu_cxx::__concurrence_unlock_error::what'/>
